ivy/functional/frontends/numpy/mathematical_functions/floating_point_routines.py

```diff
@@ -28,4 +29,26 @@ def _nextafter(
     dtype=None,
     subok=True,
 ):
-    return ivy.nextafter(x1, x2, out=out)
+    return ivy.nextafter(x1, x2, out=out) # Return result
+
+
+
+# This is the spacing function.
+# It returns the distance between x and the nearest adjacent number.
+@handle_numpy_out
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+@handle_numpy_casting
+@from_zero_dim_arrays_to_scalar
+def _spacing(
+    x,
+    /,
+    out=None,
+    *,
+    where=True,
+    casting="same_kind",
+    order="K",
+    dtype=None,
+    subok=True,
+):
+    return ivy.spacing(x, out=out) # Return result
```

Comment:
the spacing function is not yet available in the ivy functional API, please either implement this frontend function through compositions, or in other cases, propose the function to be implemented in the ivy API, more info here: https://unify.ai/docs/ivy/overview/contributing/open_tasks.html#ivy-experimental-api

Reply:
Thanks @juliagsy !  In place of

`return ivy.spacing(x, out=out)`

I replaced it with    `return ivy.Array(x, out=out)` .

Is that ok?